eras/shelley/formal-spec/utxo.tex

```diff
@@ -401,9 +401,9 @@ \subsection{Deposits and Refunds}
     & \text{total deposits for transaction} \\
     & \totalDeposits{pp}{poolParams}{certs} = \\
     &  ~~~ (\fun{keyDeposit}~pp)\cdot|\var{certs}\cap\DCertRegKey| \\
-    &  ~~~~~ + (\fun{poolDeposit}~pp)\cdot|\var{newPools}| \\
+    &  ~~~~~ + (\fun{poolDeposit}~pp)\cdot|\{\cwitness{c} ~\mid c\in~\var{newPools}\}| \\
     & ~~~\where \\
-    &  ~~~~~~~ \var{newPools} = \{\var{certs}\cap\DCertRegPool ~\mid~ \cwitness{c}\notin \var{poolParams}\}
+    &  ~~~~~~~ \var{newPools} = \{c ~\mid~ \var{certs}\cap\DCertRegPool,\cwitness{c}\notin \var{poolParams}\}
```

Comment:
I don't understand this line. Is `c` supposed to be an element of `certs ∩ DCertRegPool`? The old version doesn't make sense to me either btw.

Reply:
and the extra `c \in` totally breaks the table spacing/corners now :laughing: :sob: